package_linter.py

```diff
@@ -132,7 +132,8 @@ def check_script(path, script_name):
     if check_file_exist(script_path) == 0:
         return
     print (c.BOLD + c.HEADER + "\n>>>>",
-           scripts[i].upper(), "SCRIPT <<<<" + c.END)
+           script_name.upper(), "SCRIPT <<<<" + c.END)
```

Comment:
Thanks. Why it is working well?
It is a python cache?
Or the variable could be used on many function?